views/js/e2e/_helpers/commands/setupCommands.js

```diff
@@ -79,7 +79,7 @@ Cypress.Commands.add('publishTest', (testName) => {
 
     // Select tree node
     cy.get(setupSelectors.testsPage.rootTestClass).within(() => {
-        cy.contains(testName).click({ force: true });
+        cy.contains(testName).click();
```

Comment:
I believe we may have used `force: true` here because the list item for your desired test can be off screen, depending on the size of the viewport (and number of other tests).

Reply:
ok. I will add again.